### ISSUE TITLE
Documentation request: Instructions how to backup existing firmware

### ISSUE THREAD

**Issue by dsample** [openipcamera/openipc-firmware#122]
The Wyze Cam seems to be US a rebadged Hualai camera. Neos Smartcam is a similarly rebadged camera in the UK.

In order to be able to revert to original firmware, could process used to extract the original firmware be documented? Those who purchase a different brand (configured to connect to a different service, etc.) could contribute original firmwares for recovery use.

**Comment by loglux** [comment]
Are you asking about Neos Smartcam firmware?

**Comment by dsample** [comment]
> Are you asking about Neos Smartcam firmware?

Yes. The Neos app isn't bad, but I'd like to try out the openipc firmware. If there would be a way to get back to the original firmware, that would be useful.

**Comment by loglux** [comment]
Take it
https://github.com/loglux/Neos-SmartCam
